### ISSUE TITLE
Console is not showing connections

### ISSUE THREAD

**Issue by hguerrero** [skupperproject/skupper#347]
When connecting 2 sites, the console is not showing the arrows showing the connections anymore. 

![image](https://user-images.githubusercontent.com/1001939/102411393-df255000-3fbf-11eb-8ff1-a35a69ff0122.png)

**Comment by grs** [comment]
What skupper version? Can you get the output for <console-url>/DATA?

**Comment by hguerrero** [comment]
Version is skupper 0.4.0

`/DATA`

```json
{
    "sites": [
        {
            "site_name": "azure",
            "site_id": "218dc1ab-ba41-4c9c-b9ce-986aacb9d01d",
            "connected": [],
            "namespace": "azure",
            "url": "52.151.225.245",
            "edge": false
        },
        {
            "site_name": "local",
            "site_id": "03815489-81a4-492e-97a0-68470ff2df58",
            "connected": [
                "218dc1ab-ba41-4c9c-b9ce-986aacb9d01d"
            ],
            "namespace": "local",
            "url": "",
            "edge": true
        }
    ],
    "services": [
        {
            "address": "hello-world-backend",
            "protocol": "tcp",
            "targets": [
                {
                    "name": "172.17.0.8",
                    "target": "hello-world-backend",
                    "site_id": "03815489-81a4-492e-97a0-68470ff2df58"
                }
            ]
        }
    ]
}
```

**Comment by grs** [comment]
Should be fixed in 0.4.1 and on master
